exchanges/btse/btse_wrapper.go

```diff
@@ -1077,77 +1077,54 @@ func (e *Exchange) GetMarketSummary(ctx context.Context, symbol string, spot boo
 
 // GetFuturesContractDetails returns details about futures contracts
 func (e *Exchange) GetFuturesContractDetails(ctx context.Context, item asset.Item) ([]futures.Contract, error) {
-	if !item.IsFutures() {
-		return nil, futures.ErrNotFuturesAsset
-	}
 	if item != asset.Futures {
 		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, item)
 	}
 	marketSummary, err := e.GetMarketSummary(ctx, "", false)
 	if err != nil {
 		return nil, err
 	}
-	resp := make([]futures.Contract, 0, len(marketSummary))
+	resp := make([]futures.Contract, len(marketSummary))
 	for i := range marketSummary {
-		var cp currency.Pair
-		cp, err = currency.NewPairFromStrings(marketSummary[i].Base, marketSummary[i].Symbol[len(marketSummary[i].Base):])
+		cp, err := currency.NewPairFromStrings(marketSummary[i].Base, marketSummary[i].Symbol[len(marketSummary[i].Base):])
```

Comment:
🐒 Optional nitpick:
I think that this is too painful to do inline and without explaining that .Quote currency is the settlement currency, and do something more like:
```suggestion
		// .Quote is the settlement currency; Get the real quote currency from Symbol
		quoteCurr := marketSummary[i].Symbol[len(marketSummary[i].Base):]
		cp, err := currency.NewPairFromStrings(marketSummary[i].Base, quote)
```